packages/grid/x-grid/src/tests/filtering.XGrid.test.tsx

```diff
@@ -0,0 +1,175 @@
+import { ApiRef, FilterModel, LinkOperator, useApiRef, XGrid } from '@material-ui/x-grid';
+import { expect } from 'chai';
+import * as React from 'react';
+import { getColumnValues, sleep } from 'test/utils/helperFn';
+import { createClientRenderStrictMode } from 'test/utils';
+
+describe('<XGrid /> - Filter', () => {
+  // TODO v5: replace with createClientRender
+  const render = createClientRenderStrictMode();
+
+  before(function beforeHook() {
+    if (/jsdom/.test(window.navigator.userAgent)) {
+      // Need layouting
+      this.skip();
+    }
+  });
+
+  let apiRef: ApiRef;
+
+  const TestCase = (props: { rows?: any[]; model: FilterModel }) => {
+    const baselineProps = {
+      rows: [
+        {
+          id: 0,
+          brand: 'Nike',
+        },
+        {
+          id: 1,
+          brand: 'Adidas',
+        },
+        {
+          id: 2,
+          brand: 'Puma',
+        },
+      ],
+      columns: [{ field: 'brand' }],
+    };
+
+    const { model, rows } = props;
+    apiRef = useApiRef();
+    return (
+      <div style={{ width: 300, height: 300 }}>
+        <XGrid
+          apiRef={apiRef}
+          {...baselineProps}
+          rows={rows || baselineProps.rows}
+          filterModel={model}
+          disableColumnFilter={false}
+        />
+      </div>
+    );
+  };
+
+  const renderBrandContainsA = () => {
+    const model = {
+      items: [
+        {
+          columnField: 'brand',
+          value: 'a',
+          operatorValue: 'contains',
+        },
+      ],
+    };
+
+    render(<TestCase model={model} />, { strict: false });
```

Comment:
There are two tests that pass with `strict: true` (the default) but that run with `strict: false` because of this line.

Could you duplicate the render calls and remove `strict: false` in these two cases so we can run as many tests as possible in strict mode?